### ISSUE TITLE
Loss stays at 0 during training

### ISSUE THREAD

**Issue by hh2712** [MhLiao/TextBoxes#79]
Hi @MhLiao ,
I am attempting to finetuning textboxes on a different dataset. 

I have managed to generate training data and test data to lmdb format from create_data.sh + create_list.sh, and started the training, with everything untouched(all the network configuration, except making batch size to 4 and size of the image to 700 by 700). 

I spotted that the loss stays at 0 (*1 = 0) during training period, which obviously doesn't make sense.

Weights and dataset(training and validation) should be loaded correctly, otherwise Caffe would have complained.  

Could you please kindly provide me with any insights in debugging this problem?

Thank you in advance.

Regards,
Hanyi Hu

**Comment by MhLiao** [comment]
I guess there are some problems in your ".xml" ground truth. Can you show me one ".xml" file of your dataset?

**Comment by hh2712** [comment]
Hi @MhLiao,
Thank you for your swift reply.

I have attached one of my sample .xml ground truth label and converted it into .txt format, since github doesn't support .xml file. 
[IMG_0059.txt](https://github.com/MhLiao/TextBoxes/files/1588123/IMG_0059.txt)

Regards,
Hanyi Hu

**Comment by MhLiao** [comment]
@hh2712 Maybe you ignored the "<name>text</name>" in your ".xml" files, which caused the problem.

**Comment by hh2712** [comment]
Does the xml file have to be pretty printed? Though I did use lxml.etree.tostring function to pretty printed, but the function could fail with some corner cases.

**Comment by hh2712** [comment]
You were right. I got it working(Loss is decreasing). Cheers.
